CS3219-AY2425S1/cs3219-ay2425s1-project-g11
# Add live collaboration between users

To test locally:
1. Run docker-compose up --build
2. Open an incognito browser and a normal browser
3. Login to different accounts on both browsers
4. Match both users using matching feature
5. Type in the code editor and observe

O ya rmb to get the updated env from our drive
